src/views/ont/OntReception.vue

```diff
@@ -27,50 +31,53 @@ import Alert from '@/components/Alert'
 import Helper from '@/mixins/Helper'
 import { getPlates, transformPlates} from '@/api/SequencescapePlates'
 import CREATE_PLATE_WITH_SAMPLES from '@/graphql/queries/CreatePlateWithSamples.mutation.gql'
+import Spinner from 'vue-simple-spinner'
+import Api from '@/mixins/Api'
 
 export default {
   name: 'Reception',
-  mixins: [Helper],
-  props: {
-  },
+  mixins: [Helper, Api],
   data () {
     return {
       barcodes: [],
-      plates: {}
+      busy: false
     }
   },
   components: {
-    Alert
+    Alert,
+    Spinner
   },
   methods: {
-    async getSequencescapePlates (barcodes) {
-      return await getPlates(this.$store.getters.api.sequencescape.plates, barcodes)
-    },
-    getBarcodes () {
-      return this.barcodes.split('\n').filter(Boolean).join(',')
+    async getSequencescapePlates() {
+      await getPlates(this.sequencescapeRequest, this.formattedBarcodes)
     },
-    async handleSequencesapePlates () {
-      let jsonPlates = await this.getSequencescapePlates(this.getBarcodes())
-      if (jsonPlates !== undefined) {
-        this.plates = transformPlates(jsonPlates)
+    async handleCreateTractionPlates () {
+      this.busy = true
+      let ssPlatesJson = await this.getSequencescapePlates()
+
+      if (ssPlatesJson.length === 0) {
```

Comment:
I think this should be 
`if (ssPlatesJson.length === undefined) {`

Reply:
Is it possible to add a test for it? You could add a test to say that when ssPlatesJson is undefined it sets busy to false.

Reply:
Adding the return to the above returns an empty array, so will keep the condition the same